Gemfile

```diff
@@ -26,6 +26,7 @@ group :development, :test do
   gem "byebug", platforms: %i[mri mingw x64_mingw]
   gem "capybara"
   gem "factory_bot_rails"
+  gem "pry"
```

Comment:
I know we've got byebug already, but I think it's nice to have both as lots of developers seem pretty evenly split between preferring one or the other!